package com.cognizant.authentication.service.exceptions;

public class UserAlreadyExistsException extends Exception {
	public UserAlreadyExistsException(){
		System.out.println("User Alresy Exists");
	}
}
